### ISSUE TITLE
Update Angular version

### ISSUE THREAD

**Issue by devpato** [contentful-labs/product-catalogue-web.ts#11]
Project has great examples for a more advance use of contentful. Updating to the latest version of Angular is recommended.

**Comment by v1shaL-b** [comment]
Yeah. As there are a lot of changes and it seems sometimes you can't display the content using async pipe.
